adage/adage/settings.py

```diff
@@ -23,10 +24,17 @@
     CONFIG = CI_CONFIG
     pg_user = os.environ.get('PG_USER')
     pg_pass = os.environ.get('PG_PASSWORD')
-    CONFIG['dbmaster']['USER'] = pg_user
-    CONFIG['dbmaster']['PASSWORD'] = pg_pass
```

Comment:
Thank you for cleaning out my cruft, and sorry I left it in there! I got Codeship running more easily than I expected and I never went back and removed this.